content/200-concepts/200-database-connectors/03-postgresql.mdx

```diff
@@ -82,6 +82,7 @@ The following arguments can be used:
 | `host`             | No       |                        | Points to a directory that contains a socket to be used for the connection                                                                                                        |
 | `socket_timeout`   | No       |                        | Maximum number of seconds to wait until a single query terminates                                                                                                                 |
 | `pgbouncer`        | No       | `false`                | Configure the Engine to [enable PgBouncer compatibility mode](/guides/performance-and-optimization/connection-management/configure-pg-bouncer/)                                   |
+| `application_name` | No       |                        | Specifies a value for the application_name configuration parameter 
 
 As an example, if you want to connect to a schema called `myschema`, set the connection pool size to `5` and configure a timeout for queries of `3` seconds, you can use the following arguments:
```

Comment:
I don't think this change is correct. Everything in this sentence belongs together and forms on unit:
> As an example, [...], you can use the following arguments: [...]
(Not sure about the commas though)

Reply:
Ah, so the "if you want to" phrase relates to three things: 1. connect to `myschema`, 2. set the connection pool size, and 3) configure the time out. Got it, then we need to make more clear by something like this:

As an example, use the following arguments to connect to a schema called `myschema`, set the connection pool size to `5`, and then configure a timeout for queries of `3` seconds:

The original way was quite hard to parse, imo, because the phrase "set the connection pool size..." was ambiguous, unclear if it was the imperative to do that task, or one of the goals. ;-)

Reply:
Yes, the new suggestions sounds correct to me.